packages/hoppscotch-sh-admin/src/composables/useConfigHandler.ts

```diff
@@ -255,7 +259,21 @@ export function useConfigHandler(updatedConfigs?: Config) {
     return config;
   });
 
-  // Trasforming the working configs back into the format required by the mutations
+  // Checking if any of the config fields are empty
+  const isFieldEmpty = (field: string) => field === '';
+
+  const AreAnyConfigFieldsEmpty = (config: Config): boolean => {
+    const providerFieldsEmpty = Object.values(config.providers).some(
+      (provider) => [...Object.values(provider.fields)].some(isFieldEmpty)
+    );
+    const mailFieldsEmpty = [...Object.values(config.mailConfigs.fields)].some(
```

Comment:
Isn't it redundant to create another array since `Object.values` already return a string array?

```suggestion
      (provider) => Object.values(provider.fields).some(isFieldEmpty)
    );
    const mailFieldsEmpty = Object.values(config.mailConfigs.fields).some(
```